.gitignore

```diff
@@ -15,6 +15,7 @@ dist
 # testing
 /coverage
 .vscode-test-web
+.vscode-test
```

Comment:
I would siggest to add .vscode-test into .vscodeignore to not bundle into .vsix package

Reply:
I was lucky, it is already set in .vscodeignore https://github.com/KaotoIO/vscode-kaoto/blob/1393a8a8d3351ee58e920dcabebcb9eb5451260f/.vscodeignore#L2